### PR TITLE
Added to the About page.

### PR DESCRIPTION
**Pull request by Withington**
I've tried to write an intro for a lay audience. Perhaps the first section is a bit repetitive? I didn't want to remove your project aims. See what you think. Maybe separate pages are needed - for clinicians, IRD-affected people and people involved/interested in the project itself?

### CODE REVIEW

File: views/about.slim [hunk 1]
```diff
@@ -3,11 +3,59 @@
     h1.light-blue-text.text-darken-3 About
     .card
       .card-content
+        h2 What is Eye2Gene?
+        ol
```
Comment (IsmailM) [inline views/about.slim]: why the extra `ol`?  

Essentially these would be empty tags....

File: views/about.slim [hunk 2]
```diff
@@ -3,11 +3,59 @@
...
+        p Eye2Gene is an app that will help clinicians provide a genetic diagnosis for people with progressive sight-loss caused by an inherited retinal dystrophy (IRD).
+        ol
+        p Over 180 gene mutations are known to cause IRD and a genetic diagnosis is a significant step towards managing, and possibly treating, people's sight-loss.
+        ol
+        p Eye2Gene will make genetic diagnosis faster and accessible to more IRD-affected families. Eye2Gene will be trained using the data of thousands of patients so that it can learn to distinguish between the different genetic causes. In the first stage of development, Eye2Gene will be trained using auto-fluorescent images of retinas. You will be able to upload a retina image and Eye2Gene will report which genes are most likely to be causing the IRD.
+        ol
+        h3. Aims
         p.lead The aims of Eye2Gene are to:
         ol
```
Comment (IsmailM) [inline views/about.slim]: This is the only place in the document where an `ol` tag is needed

**Comment by Withington** [comment]
Hi Ismail. This is the first time I've used slim and I found it difficult to get the information I needed about it. I wanted blank lines between paragraphs and ol seemed to be how to do it. Please advise! :-)

**Comment by IsmailM** [comment]
So that was a CSS issue - see my latest commit on this branch...

Happy to merge this now 👍 

A few other random stuff:

We use SCSS - which is basically CSS but allows for variables and nested CSS....

BUT it needs to be compiled before it can be run:

```
# u may need to install sass and uglifyjs
# gem install sass
# npm install -g uglifyjs
bundle exec assets
```
Also running Eye2Gene with the debug command will make life easier when changing any static stuff...

```
bundle exec eye2gene -n 4 -D
```

The public folder is hosted from ~/.eye2gene/public (which is safer in production). However, this folder is generated on start up and contains a copy of the public folder etc. With the debug command, it creates a soft link to the public folder in the dir... (so in theory you don't need to restart every time...


Also, with regards to SLIM: this may be useful https://html2slim.herokuapp.com/. Once you get your head around SLIM, it makes a lot more sense than regular HTML - most web developers use some preprocessor of some variant (e.g. HAML, jade etc.)... Moreover, It compresses the HTML output on compilation which is simply better for production - and means you have to type fewer lines which is always good 👍

File: views/about.slim [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
+#about_container.container
   .section
     h1.light-blue-text.text-darken-3 About
     .card
       .card-content
+        h2 What is Eye2Gene?
+        p Eye2Gene is an app that will help clinicians provide a genetic diagnosis for people with progressive sight-loss caused by an inherited retinal dystrophy (IRD).
+        p Over 180 gene mutations are known to cause IRD and a genetic diagnosis is a significant step towards managing, and possibly treating, people's sight-loss.
+        p Eye2Gene will make genetic diagnosis faster and accessible to more IRD-affected families. Eye2Gene will be trained using the data of thousands of patients so that it can learn to distinguish between the different genetic causes. In the first stage of development, Eye2Gene will be trained using auto-fluorescent images of retinas. You will be able to upload a retina image and Eye2Gene will report which genes are most likely to be causing the IRD.
+        h4. Aims
```
[the remark this answers is not in the document]
Comment (Withington) [reply views/about.slim]: Agreed, they did look too big.

**Comment by Withington** [comment]
Thanks Ismail

**Comment by pontikos** [comment]
Thanks @Withington !  Is this live @IsmailM ?

**Comment by IsmailM** [comment]
It will be tonight. 👍


via Newton Mail [https://cloudmagic.com/k/d/mailapp?ct=pa&cv=9.8.376&pv=8.1.0&source=email_footer_2] On Mon, Jul 23, 2018 at 10:36am, Nikolas Pontikos < notifications@github.com [notifications@github.com] > wrote:
Thanks @Withington [https://github.com/Withington] ! Is this live @IsmailM [https://github.com/IsmailM] ?

**Comment by pontikos** [comment]
Thanks Ismail!  I had written a PPI section too, could we make that visible
too please without login?

**Pull request merged**